src/test/java/hudson/plugins/buildblocker/BlockingJobsMonitorTest.java

```diff
@@ -75,65 +81,180 @@ protected void setUp() throws Exception {
         while (!slave.getComputer().getExecutors().get(0).isBusy()) {
             TimeUnit.SECONDS.sleep(1);
         }
+    }
+
+    protected void WorkFlowSetUp() throws Exception {
+        blockingJobName = "blockingJob";
+
+        // clear queue from preceding tests
+        Jenkins.getInstance().getQueue().clear();
```

Comment:
BTW this is probably pointless: when using `@Rule JenkinsRule`, each test case (method) gets its own fresh `Jenkins` instance.